frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwarePackageCard/SoftwarePackageCard.tsx

```diff
@@ -261,9 +273,33 @@ const SoftwarePackageCard = ({
       FileSaver.saveAs(file);
     } catch (e) {
       console.log(e);
-      renderFlash("error", "Couldn’t download. Please try again.");
+      renderFlash("error", "Couldn't download. Please try again.");
```

Comment:
Nit: apostrophe formatting

Reply:
sorry what do you mean here. It was changes due to VS code warning about ambiguous characters.

Reply:
The new character is the straight apostrophe. Don't we use the slanted one instead?

Reply:
ah hum interesting. ok I think i want to look into this more. I was changing it cause vs code was giving me warnings, but it seems in figma we are using slanted apostrophe and that's how it comes in when copy and pasting. I'm gonna look at this a bit more to see if we should end up changing these or just leaving how it comes from figma and changing the VS code settings.